tinkerpop3/src/test/java/io/shiftleft/overflowdb/IndexesTest.java

```diff
@@ -25,7 +23,7 @@ public void shouldUseIndices() throws IOException {
 
     { // tests with index
       OdbGraph graph = GratefulDead.newGraphWithData();
-      graph.createIndex("performances", Vertex.class);
+      graph.indexManager.createNodePropertyIndex("performances");
```

Comment:
Not sure if indexes are used somewhere in dependent projects, maybe this will require additional renaming? Otherwise looks OK

Reply:
yes i think they are, will provide follow up PRs on those